### PR TITLE
Ik 3551 unit test coverage cmd happy paths

### PR DESCRIPTION
**Pull request by martasd**
# Checklist

- [x] `make test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added

## Description of change

Adds tests for success in calling functions from cmd modules.

Fixes indykite/jarvis-backlog#301

### CODE REVIEW

File: tests/test_change_password.py [hunk 1]
```diff
@@ -85,3 +87,22 @@ def test_password_of_user_nonexisting_twin_id(capsys):
...
+
+
+def test_password_of_user_success(capsys):
+    digital_twin_id = "e1e9f07d-fc6e-4629-84d1-8d23836524ba"
+    tenant_id = data.get_tenant()
+    password = data.get_new_password()
+
+    with patch("jarvis_sdk.cmd.IdentityClient") as client:
+        client.stub.ChangePassword.return_value = pb2.ChangePasswordResponse()
+        response = client.change_password_of_user(digital_twin_id, tenant_id, password)
```
Comment (martasd) [inline tests/test_change_password.py:99]: Do you have any advice about how to correctly mock this function's response?

File: tests/test_delete.py [hunk 2]
```diff
@@ -45,6 +46,18 @@ def test_del_digital_twin_nonexisting_twin_id(capsys):
     assert response is None
 
 
+# Currently, this test would delete existing digital twin in Jarvis Dev.
```
Comment (maprokes) [inline tests/test_delete.py:49]: And it will be enabled when you finish the grpc server mocking right?

**Review by maprokes — COMMENTED** [review]
@martasd I'm not sure what would be the impact of deleting the `__init__.py` files for the package build. I remember that we had some troubles when you try to import the package from pypi in another project, it throws some import errors. Though this might be related to the python version as well. Could you try to build the package and import it and use it in another project? Maybe @kilimandzsaro could help with that as well?

File: tests/test_delete.py (hunk 2 above)
Comment (martasd) [reply tests/test_delete.py:49]: Yeah, that's where the mocking is needed, so that existing data would not be deleted.

**Comment by martasd** [comment]
> @martasd I'm not sure what would be the impact of deleting the `__init__.py` files for the package build. 

Not sure how those actually got deleted. I reverted back those deletions- I don't think they should be deleted.

**Comment by kilimandzsaro** [comment]
Which customer will use this sdk? Is it ok them to use python 3.10?

**Comment by martasd** [comment]
> Which customer will use this sdk? Is it ok them to use python 3.10?

I don't know whether anyone is using the SDK at the moment. I will ask Elisabeth about that.

**Comment by martasd** [comment]
> Which customer will use this sdk? Is it ok them to use python 3.10?

@kilimandzsaro We decided that it's sensible not to change the Python version if not necessary, so I reverted back to 3.8.

**Review by maprokes — APPROVED** [review]

**Pull request merged**